docs/manual/forms/form_fields.rst

```diff
@@ -17,17 +17,58 @@ Basic
 -----
 
 * **Label**: Het label bij het veld dat zichtbaar is voor de eindgebruiker.
-* **Property Name**: De interne naam van het veld. Deze naam wordt gebruikt om 
+* **Eigenschapnaam**: De interne naam van het veld. Deze naam wordt gebruikt om
   eenduidig naar dit veld te verwijzen vanuit andere velden of in 
   :ref:`sjablonen <manual_templates>`.
-* **Description**: Een optionele help tekst bij het veld.
-* **Show in email**: Indien aangevinkt, dan wordt dit veld opgenomen in de 
-  bevestigingse-mail naar de eindgebruiker.
-* **Hidden**: Indien aangevinkt, dan is het veld onzichtbaar voor de 
+* **Beschrijving**: Een optionele help tekst bij het veld.
+* **Weergeven in bevestigingsmail**: Indien aangevinkt, dan wordt dit veld opgenomen in de
+  bevestigingsmail naar de eindgebruiker.
+* **Verborgen**: Indien aangevinkt, dan is het veld onzichtbaar voor de
   eindgebruiker. Dit kan bijvoorbeeld gebruikt worden om informatie voor in te 
   vullen en door te zetten naar achterliggende registratiesystemen. Beheerders
   kunnen de waarden van onzichtbare velden uiteraard wel zien.
 
+Variabelen
+~~~~~~~~~~
+
+.. todo:: add reference to vars documentation (see
+   https://github.com/open-formulieren/open-forms/pull/1893)
+
+U kunt variabelen gebruiken in het ontwerp van formulieren, bijvoorbeeld om een
+(voor-)ingevulde naam in het label van een ander formulierveld weer te geven. Hiervoor
+is de :ref:`sjabloonfunctionaliteit <manual_templates>` beschikbaar.
+
+.. note::
+
+    Alle velden in een formulier zijn als variabele beschikbaar. Daarnaast zijn er ook
+    altijd een aantal standaardvariabelen beschikbaar én u kunt zelf ook variabelen
```

Comment:
In the variable docs I think we referred to these as 'vaste variabelen'? Not sure if its worth mentioning here instead of standaardvariabelen

Reply:
yeah, I'll update this PR :-) consistency is better